### PR TITLE
feat: new distro v1.22.0, new versioning and fix SA issue

### PR DESCRIPTION
**Pull request by nutellinoit**

### CODE REVIEW

File: docs/releases/v1.22.0.md [hunk 1]
```diff
@@ -1,50 +1,42 @@
...
+different versioning schema. KFD version now will closely follow the version of
+the latest `Kubernetes` release that is supported by Fury. This release supports `kubernetes` runtime `v1.22.0`.
 
 This distribution is maintained with ❤️ by the team [SIGHUP](https://sighup.io/),
 and is battle tested in production environments.
 
+This new release of KFD features a bundle of all the core modules supported by
+Fury with enhancements and bug fixes. Alongside, Fury gets a new face with this
+release. We would like to introduce our brand new
+[documentation site](https://docs.kubernetesfury.com/) with this release.
```
Comment (ralgozino) [inline docs/releases/v1.22.0.md:13]:
```suggestion
This new release of KFD features a bundle of all the core modules supported by
Fury with enhancements and bug fixes.
```

File: docs/releases/v1.22.0.md [hunk 2]
```diff
@@ -1,50 +1,42 @@
...
+- [monitoring](https://github.com/sighupio/fury-kubernetes-monitoring) 📦 core module: [**v1.14.2**](https://github.com/sighupio/fury-kubernetes-monitoring/releases/tag/v1.14.2)
+  No update.
+- [logging](https://github.com/sighupio/fury-kubernetes-logging) 📦 core module: [**v1.10.3**](https://github.com/sighupio/fury-kubernetes-logging/releases/tag/v1.10.3)
+  No update.
+- [ingress](https://github.com/sighupio/fury-kubernetes-ingress) 📦 core module: [**v1.12.2**](https://github.com/sighupio/fury-kubernetes-ingress/releases/tag/v1.12.2)
+  No update.
+- [dr](https://github.com/sighupio/fury-kubernetes-dr) 📦 core module: [**v1.9.0**](https://github.com/sighupio/fury-kubernetes-dr/releases/tag/v1.9.0)
+  No update.
+- [OPA](https://github.com/sighupio/fury-kubernetes-opa) 📦 core module: [**v1.6.0**](https://github.com/sighupio/fury-kubernetes-opa/releases/tag/v1.6.0)
+  No update.
```
[the remark this answers is not in the document]
Comment (nutellinoit) [reply docs/releases/v1.22.0.md:31]: Against 1.21.0
The idea is that 1.7.0->1.21.0->1.22.0->1.23.2

File: docs/releases/v1.22.0.md [hunk 3]
```diff
@@ -1,50 +1,40 @@
...
-fixing bugs in all the core modules.
-
-The team has been working to make the release upgrade as simple as possible, so read carefully the upgrade path of each
-core module listed below along with the upgrade path of the distribution.
-
-⚠️ If upgrading from `v1.23.0`, you **must** delete all the objects (StatefulSet, Deployment, DaemonSet, etc) as specified in the release notes of the modules
-**before** upgrading to `v1.23.1`.
+Welcome to the KFD release `v1.22.0`. From this release on, Fury follows a
+different versioning schema. KFD version now will closely follow the version of
+the latest `Kubernetes` release that is supported by Fury. This release supports `kubernetes` runtime `v1.22.0`.
```
Comment (ralgozino) [inline docs/releases/v1.22.0.md:5]:
```suggestion
the latest `Kubernetes` release that is supported by Fury. This release supports `kubernetes` runtime `v1.22.x`.
```

**Review by ralgozino — APPROVED** [review]
very subtle correction, otherwise lgtm 🚀

**Comment by nutellinoit** [comment]
This PR should be good to go

**Pull request merged**